_includes/partners-list.html

```diff
@@ -0,0 +1,30 @@
+<center class='each-partner'>
+  <div class="partner-post">
+    <img src="../img/cure-logo.png" class="partner-logo" >
+  </div>
+  <h3 class="partner-name">Cure International</h3>
+</center>
+<center class='each-partner'>
```

Comment:
The `<center>` should not be used, as it was a presentational element and [has been deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) from the HTML spec.